src/main/resources/application.properties.template

```diff
@@ -9,7 +9,7 @@ spring.datasource.driver-class-name=org.postgresql.Driver
 
 spring.jpa.hibernate.ddl-auto=create-drop
 spring.jpa.show-sql=false
-spring.jpa.properties.hibernate.hbm2ddl.import_files=sql/crops.sql, sql/lists.sql, sql/locations.sql, sql/people.sql, sql/programs.sql, sql/trials.sql, sql/seasons.sql, sql/studies.sql, sql/breeding_methods.sql, sql/germplasm.sql, sql/attribute_defs.sql, sql/attribute_values.sql, sql/seed_lots.sql, sql/observation_units.sql, sql/crosses.sql, sql/pedigree.sql, sql/events.sql, sql/images.sql, sql/observation_variables.sql, sql/observations.sql, sql/samples.sql, sql/allele_calls.sql, sql/genome_maps.sql, sql/references.sql, sql/vendor.sql    
+spring.jpa.properties.hibernate.hbm2ddl.import_files=sql/crops.sql, sql/lists.sql, sql/locations.sql, sql/people.sql, sql/programs.sql, sql/trials.sql, sql/seasons.sql, sql/studies.sql, sql/breeding_methods.sql, sql/germplasm.sql, sql/attribute_defs.sql, sql/attribute_values.sql, sql/seed_lots.sql, sql/observation_units.sql, sql/crosses.sql, sql/pedigree.sql, sql/events.sql, sql/images.sql, sql/observation_variables.sql, sql/observations.sql, sql/samples.sql, sql/allele_calls.sql, sql/genome_maps.sql, sql/references.sql, sql/vendor.sql, sql/create_indexes.sql
```

Comment:
I think we should also update the application.properties files in the bi-api and bi-docker-stack repos so they will run the create_indexes.sql for new deployments

Reply:
Seems like a good idea, would create_indexes.sql need to be added to those repos as well? I only see species.sql in bi-api and bi-docker-stack.

Reply:
Thanks. I've made the change in https://github.com/Breeding-Insight/bi-api/pull/283 and https://github.com/Breeding-Insight/bi-docker-stack/pull/26. Still doing some testing, I'll request reviews on those soon.